Xamarin.Forms.Core/Label.cs

```diff
@@ -78,6 +78,8 @@ public class Label : View, IFontElement, ITextElement, ITextAlignmentElement, IL
 
 		public static readonly BindableProperty LineHeightProperty = LineHeightElement.LineHeightProperty;
 
+		public static readonly BindableProperty MaxLinesProperty = BindableProperty.Create(nameof(MaxLines), typeof(int), typeof(Label), -1);
```

Comment:
we probably should InvalidateMeasure in case the property changes.

also, there's some code in `OnTextChanged()` that rely on some heuristics to find out if the label is single line or not. those heuristics aren't probably valid anymore.